src/components/Highlight/index.tsx

```diff
@@ -26,6 +26,10 @@ const Highlighted = styled.span`
  * Component will render a SPAN or series of SPAN with the content and highlights
  */
 export const Highlight: React.FC<HighlightProps> = ({ text, highlight, ignoreCase = true, theme }) => {
+  if (highlight === '') {
+    return <span>{text}</span>;
```

Comment:
why not just `return text;`?

Reply:
Types, I'm retuning react elements and should be consistent, so can't just return a String here.

Reply:
Right, wouldn't be this better instead of wrapping by span `return <>{text}</>`;?

Reply:
Unfortunately, that's [the issue](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/20544) of typings. React actually allows returning the string.